pkg/pod/container.go

```diff
@@ -235,6 +235,25 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMounts configures pod volumes to mount inside the container.
+func (builder *ContainerBuilder) WithVolumeMounts(volMounts []v1.VolumeMount) *ContainerBuilder {
```

Comment:
Can you change this to ` WithVolumeMount` and just accept a single v1.VolumeMount? Multiple mounts can be chained together.

Reply:
resolved in #182